tables/atom.py

```diff
@@ -1120,25 +1231,19 @@ class VLUnicodeAtom(_BufferedAtom):
     # NumPy scalars inherit the UCS-2 encoding from Python (see
     # NumPy ticket #525).  Since ``_tobuffer()`` can't return an
     # array, we must override ``toarray()`` itself.
-    def toarray(self, object_):
-        if isinstance(object_, bytes):
-            warnings.warn("Storing bytestrings in VLUnicodeAtom is "
-                          "deprecated.", DeprecationWarning)
-        elif not isinstance(object_, str):
+    def toarray(self, object_: str):
+        if not isinstance(object_, str):
             raise TypeError(f"object is not a string: {object_!r}")
         ustr = str(object_)
         uarr = np.array(ustr, dtype='U')
         return np.ndarray(
             buffer=uarr, dtype=self.base.dtype, shape=len(ustr))
 
-    def _tobuffer(self, object_):
+    def _tobuffer(self, object_: str):
         # This works (and is used) only with UCS-4 builds of Python,
         # where the width of the internal representation of a
         # character matches that of the base atoms.
-        if isinstance(object_, bytes):
-            warnings.warn("Storing bytestrings in VLUnicodeAtom is "
-                          "deprecated.", DeprecationWarning)
```

Comment:
item

Reply:
Sorry, not following this comment.

Reply:
Also this change should be mentioned in the release notes.

Reply:
OK it is already mentioned together in the end in the release note item you have already added.